lib/solid_queue/scheduler.rb

```diff
@@ -35,7 +35,15 @@ def run
       end
 
       def schedule_recurring_tasks
-        recurring_schedule.schedule_tasks
+        if recurring_schedule.valid?
+          recurring_schedule.schedule_tasks
+        else
+          error_messages = recurring_schedule.invalid_tasks.map do |task|
```

Comment:
It occurs to me that if you have a record written in your DB, and then it _becomes_ invalid this will error out. E.g. your record has a legacy class name. This would prevent SQ from booting, which might be undesirable. Perhaps it's best to log the errors instead?